resources/views/components/widget.blade.php

```diff
@@ -184,18 +184,21 @@ class="cx-w-full cx-text-base cx-font-normal cx-leading-6 cx-transition-colors c
             </div>
         @endif
 
-        <div class="cx-flex cx-items-center cx-gap-6 cx-font-semibold cx-text-base {{ $this->css('cxFooter') }}">
-            @php
-                $enabledSupportLinks = SupportLink::where('enabled', true)->limit(2)->get();
-            @endphp
-            @foreach($enabledSupportLinks as $link)
-                <a
-                    href="{{ $link->link_url }}"
-                    class="cx-flex-1 cx-underline cx-capitalize {{ $loop->index == 0 ? 'cx-text-right ' : '' }} {{ $this->css('cxFooterLink') }} {{ count($enabledSupportLinks) === 1 ? 'cx-text-center' : '' }}"
-                >
-                    {{ $link->link_copy }}
-                </a>
-            @endforeach
-        </div>
+        @php
+            $enabledSupportLinks = SupportLink::where('enabled', true)->limit(2)->get();
+        @endphp
+
+        @if(empty($enabledSupportLinks))
```

Comment:
Since `$enabledSupportLinks` returns a Collection, you can't reliably use the `empty()` method (not to mention it should probably have been `! empty(...`, since it technically returns a Collection class. Instead, you can use the `->isEmpty()` or `->isNotEmpty()` methods to check if there are any items in the collection.

```php
Illuminate\Database\Eloquent\Collection {#4056 ▼
  #items: []
  #escapeWhenCastingToString: false
}
```

```suggestion
        @if($enabledSupportLinks->isNotEmpty())
```